PROJECT-CLEARWATER-MIB

```diff
@@ -1687,6 +1690,678 @@ sproutNonRegisterAuthenticationFailures OBJECT-TYPE
                  requests over the period."
     ::= { sproutNonRegisterAuthenticationEntry 4 }
 
+sproutICSCFIncomingSipTransactionsTable OBJECT-TYPE

```

Comment:
Consistency in naming - if we're capitalising one acronym (ICSCF) we should capitalise the other (SIP).